### PR TITLE
Fix grammatical issues in pull request template

### PR DESCRIPTION
**Pull request by ncovercash**
## Purpose
The current pull request template had some grammatical issues and unclear wording.  This PR resolves those errors.

## Approach
The new `PULL_REQUEST_TEMPLATE.md` uses better semantics to convey the intentions of the template.

## Refs
No applicable JIRA tickets.

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
  - No change needed as this does not affect the actual code nor releases
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [x] If not, please create them
  - [x] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [x] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [x] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [x] If so, have they been approved?

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
  How does this change fulfill the purpose? It's best to talk
  high-level strategy and avoid code-splaining the commit history.
 
- The goal is not only to explain what you did, but help other
+ The goal is not only to explain what you did, but to also help other
```
Comment (Stanislau-Snapok) [inline PULL_REQUEST_TEMPLATE.md]: How about `... but also to help ...`?

Comment (ncovercash) [reply PULL_REQUEST_TEMPLATE.md]: Sure

**Review by Stanislau-Snapok — APPROVED** [review]

**Comment by ncovercash** [comment]
Is there anything else I need to do from my end or is this okay to merge?

**Comment by Stanislau-Snapok** [comment]
> Is there anything else I need to do from my end or is this okay to merge?

You need approve from @Dmitriy-Litvinenko.

**Review by pavelspichonak — APPROVED** [review]
LGTM. Thanks!

**Comment by Stanislau-Snapok** [comment]
Thank you @ncovercash for fix grammatical issues in pull request template.
We created task https://issues.folio.org/browse/UICAL-184 associated with this problem.
We update CHANGELOG.md, squashed all commits and create new pull https://github.com/folio-org/ui-calendar/pull/361 for resolve problem with merge.

**Pull request closed without merging**